lib/eventHubClient.ts

```diff
@@ -67,15 +91,16 @@ export class EventHubClient {
   private _context: ConnectionContext;
 
   /**
-   * Instantiate a client pointing to the Event Hub given by this configuration.
+   * Instantiates a client pointing to the Event Hub given by this configuration.
    *
    * @constructor
    * @param {ConnectionConfig} config - The connection configuration to create the EventHub Client.
    * @param {TokenProvider} [tokenProvider] - The token provider that provides the token for authentication.
    * Default value: SasTokenProvider.
    */
-  constructor(config: ConnectionConfig, tokenProvider?: TokenProvider) {
-    this._context = ConnectionContext.create(config, tokenProvider);
+  constructor(config: ConnectionConfig, options?: ClientOptions) {
```

Comment:
>options?: ClientOptions) [](start = 40, length = 24)

that's a breaking change, isn't it?

Reply:
Yes. It is. I don't expect people to use the EventHubClient constructor. They should be using `EventHubClient.createFromConnectionString("...", "..")` which should take care of instantiating the client for them.